src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,54 @@
+import { IService } from '../types';
+import { DiskUsage } from 'types/motionModel';
+import { readdirSync, existsSync, mkdirSync, rename } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+import { getDateFromUnicodeTimestamp } from '../util';
+
+const diskUsage: DiskUsage = {};
+
+export const getDiskUsage = () => {
+    return diskUsage;
+};
+
+export const SaveImagesToUSB: IService = {
+    execute: async () => {
+        try {
+            const directoryPath = USB_WRITE_PATH;
+            const files = readdirSync(directoryPath);
+
+            files.forEach((file) => {
+                if (file.endsWith("jpg")) {
+                    const dateFromFileName = getDateFromUnicodeTimestamp(file);
+                    const year = dateFromFileName.getFullYear();
+                    const month = String(dateFromFileName.getMonth() + 1).padStart(2, "0"); // Add 1 because months are 0-based
+                    const day = String(dateFromFileName.getDate()).padStart(2, "0");
+
+                    // Create the formatted date string
+                    const formattedDate = `${year}-${month}-${day}`;
```

Comment:
There's also an easy way to get this from the date object like:
`new Date().toISOString().split('T')[0]`
that will give you YYYY-MM-DD

Reply:
updated!